src/parser/jobs/vpr/modules/SerpentsIre.tsx

```diff
@@ -0,0 +1,56 @@
+import {t, Trans} from '@lingui/macro'
+import {DataLink} from 'components/ui/DbLink'
+import {EvaluatedAction, ExpectedActionsEvaluator, TimedWindow} from 'parser/core/modules/ActionWindow'
+import {HistoryEntry} from 'parser/core/modules/ActionWindow/History'
+import {SEVERITY} from 'parser/core/modules/Suggestions'
+import React from 'react'
+import DISPLAY_ORDER from './DISPLAY_ORDER'
+
+const SEVERITY_TIERS = {
+	1: SEVERITY.MEDIUM,
+	2: SEVERITY.MAJOR,
+}
+
+const OPENER_BUFFER = 30000 //After 30 seconds, you have reached the point that you could do 2 reawakens if you burst right there.
+
+export class SerpentsIre extends TimedWindow { // AKA Peusdo 2 Minute Window
+	static override handle = 'serpentsIre'
+	static override title = t('vpr.serpents_ire.title')`"2 Minute Windows"`
+	static override displayOrder = DISPLAY_ORDER.SERPENTS_IRE
+
+	override startAction = this.data.actions.SERPENTS_IRE
+	override duration = 30000 // 30s. After Eye is cast, 1 GCD should be used before doing a double reawaken back to back
```

Comment:
Since this is grouching about no-magic-numbers, should probably make this be = OPENER_BUFFER if they'll always be the same amount of time, or another const if there's a chance they'll differ in the future